README.md

```diff
@@ -181,6 +181,25 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
 
 Bug reports and pull requests are welcome on GitHub at https://github.com/ruby-formatter/rufo.
 
+## Releasing a gem
+
+1. Ensure that tests pass and everything is working!
```

Comment:
Could add `the` between `that` and `tests`.

Reply:
:+1:

Reply:
Done.